packages/protobuf/README.md

```diff
@@ -1,41 +1,44 @@
 # `@penumbra-zone/protobuf`
 
-**To use this package, you need to [enable the Buf Schema Registry](https://buf.build/docs/bsr/generated-sdks/npm)**
+## If you are looking for a Penumbra extension client
+
+You should install `@penumbra-zone/client`. This package is provided for
+developers interested in lower-level work or more detailed configuration.
+
+---
+
+This package collects types and some configuration intended for use with
+`@penumbra-zone/transport-dom`.
+
+**To use this package, you need to [enable the Buf Schema Registry](https://buf.build/docs/bsr/generated-sdks/npm):**
 
 ```sh
 echo "@buf:registry=https://buf.build/gen/npm/v1/" >> .npmrc
 ```
 
-This package exports a `typeRegistry` (and inclusive `jsonOptions`) for use with
-`@bufbuild` and `@connectrpc` tooling, particularly
-`@penumbra-zone/transport-dom`.
-
-All message types necessary for a Connect `Transport` to serialize/deserialize
-communication with Prax or any other Penumbra extension are included.
+### Exports
 
-## If you simply need a Penumbra extension client
+This package exports a `typeRegistry` (and `jsonOptions` including said
+registry) for use with `createChannelTransport` or any `@connectrpc` transport.
 
-You're looking for `@penumbra-zone/client`, which handles this process for you
-and also performs some basic safety checks.
+All types necessary for a to serialize/deserialize communication with Prax or
+any other Penumbra extension are included.
 
-This package is provided for those who are interested in lower-level work or
-more detailed configuration.
+Service definitions for all relevant services are also re-exported.
 
-### Simple example
+### A Simple example
 
-```ts
+```js
 import { jsonOptions } from '@penumbra-zone/protobuf';
-
 import { createChannelTransport } from '@penumbra-zone/transport-dom';
-import type { ServiceType } from '@bufbuild/protobuf';
 
-// unsafely get first available provider
-const getPort = () => Object.values(window[Symbol.for('penumbra')])[0].connect();
+// naively get first available provider
+const provider = Object.values(window[Symbol.for('penumbra')])[0];
+void provider.request();
```

Comment:
Shouldn't they use client package for this?

Reply:
most developers will. this example is demonstrating use of the type registry this package, not the client package.

a section above this in the readme instructs, 

>If you are looking for a Penumbra extension client
>
>You should install `@penumbra-zone/client`. This package is provided for
developers interested in lower-level work or more detailed configuration.